packer/scylla_install_image

```diff
@@ -123,6 +124,7 @@ if __name__ == '__main__':
         sysconfig_opt = '--disable-writeback-cache'
         swap_opt = '--swap-directory /mnt'
 
+    run('systemctl disable --now apt-daily-upgrade.timer apt-daily.timer dpkg-db-backup.timer motd-news.timer', shell=True, check=True)
```

Comment:
If you've removed the motd-news-config package, do you still need to disable the timer?

Reply:
Although I removed this package, `notd-news.timer` is still running, since it's coming from `/etc/default/motd-news` and not from the package. so we can either disable it or modify the value `ENABLED` to `0`